src/templates/snek/gql/tasks/auth.ts

```diff
@@ -0,0 +1,142 @@
+//#region > Imports
+//> Parent Task
+// Contains the SNEK parent task
+import SnekTasks from "./index";
+//> Interfaces
+// Contains the user interface for authentication
+import { User } from "../../../../session";
+//> Types
```

Comment:
I'd suggest to insert a new line above headers to improve readability.

Reply:
The current state of our style guide has no newline between headers. So this code part is currently correct, but might be changed with an style guide update.